examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py

```diff
@@ -79,14 +85,16 @@
 ax.set_rlim(0, 1.3)
 
 plt.legend()
+plt.show()
```

Comment:
I might have removed these from more of the reproduction examples. I thought one show at the end was enough.